docs/_tooling/extensions/score_metamodel/checks/standards.py

```diff
@@ -27,7 +27,14 @@ def get_standards_needs(needs: list[NeedsInfoType]) -> dict:
     return {
```

Comment:
we have a dedicated type, don't we? Something like `.. standard-req::`. So why do we need to differentiate by id at all?